layers/+lang/purescript/config.el

```diff
@@ -12,3 +12,6 @@
 
 
 ;; Nothing here yet
+(spacemacs|defvar-company-backends purescript-mode)
+(defvar psc-ide-executable "psc-ide")
+(defvar psc-ide-server-executable "psc-ide-server")
```

Comment:
I thought that it is necessary that the user can override them in dotspacemacs-configuration-layers